gravity-sync

```diff
@@ -1465,8 +1482,14 @@ function validate_ph_folders {
         PH_FTL_CHECK=$(sudo podman container ls | grep "${PIHOLE_CONTAINER_IMAGE}")
         if [ "$PH_FTL_CHECK" == "" ]; then
             MESSAGE="${UI_VALIDATING_FAIL_CONTAINER} ${UI_CORE_APP}"
-            echo_fail
-            exit_no_change
+            echo "failed bitch"
```

Comment:
?

Reply:
Omg lol that was not supposed to be there. I'll fix it. 
Was a test for something being dumb...